internal/app/machined/pkg/controllers/k8s/render_config_static_pods.go

```diff
@@ -139,7 +160,7 @@ func (ctrl *RenderConfigsStaticPodController) Run(ctx context.Context, r control
 
 		if err = r.Modify(ctx, k8s.NewConfigStatus(k8s.ControlPlaneNamespaceName, k8s.ConfigStatusStaticPodID), func(r resource.Resource) error {
 			r.(*k8s.ConfigStatus).TypedSpec().Ready = true
-			r.(*k8s.ConfigStatus).TypedSpec().Version = admissionRes.Metadata().Version().String()
+			r.(*k8s.ConfigStatus).TypedSpec().Version = admissionRes.Metadata().Version().String() + auditRes.Metadata().Version().String()
```

Comment:
adding up versions?

Reply:
these are strings, so technically concatenating them. bumping the version here bumps the static pod manifest definition, so that kubelet reloads the pod